EPRN.Common/Dtos/JourneyAnswersDto.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace EPRN.Common.Dtos
+{
+    public class JourneyAnswersDto
+    {
+        public int JourneyId { get; set; }
+
+        public string Month { get; set; }
+
+        public string Tonnes { get; set; }
+
+        public string BaledWithWire { get; set; }
+
+        public string TonnageAdjusted { get; set; }
+
+        public string? Note { get; set; }
+
+        public string WasteType { get; set; }
```

Comment:
WasteType is being populated through an action filter now, so this field is not needed

Reply:
Not sure I understand this. Where is the action filter defined, and why would it be used as part of a record retrieval from the db?